Rubberduck.Parsing/Symbols/PropertySetDeclaration.cs

```diff
@@ -61,41 +55,38 @@ public sealed class PropertySetDeclaration : Declaration, IParameterizedDeclarat
                 null,
                 attributes)
         {
-            _parameters =
-                member.Parameters.Select(decl => new ParameterDeclaration(decl, this, module))
-                    .ToList();
+            AddParameters(member.Parameters.Select(decl => new ParameterDeclaration(decl, this, module)));
         }
 
-        public PropertySetDeclaration(ComField field, Declaration parent, QualifiedModuleName module,
-            Attributes attributes) : this(
-            module.QualifyMemberName(field.Name),
-            parent,
-            parent,
-            field.ValueType,
-            Accessibility.Global,
-            null,
-            null,
-            Selection.Home,
-            false,
-            null,
-            attributes)
+        public PropertySetDeclaration(ComField field, Declaration parent, QualifiedModuleName module, Attributes attributes) 
+            : this(
+                module.QualifyMemberName(field.Name),
+                parent,
+                parent,
+                field.ValueType,
+                Accessibility.Global,
```

Comment:
There is no `Accessibility.Global`, only `Accessibility.Public`?

Reply:
This came up several weeks earlier when we were discussing whether the scoping assigned to the built in declarations was appropriate. AFAIK these have always been hard coded to global accessibility since we started doing COM reflection, so I just preserved the original behavior when I did the refactor.  Tracking down the assumptions that are being made around this in the resolver and elsewhere is probably a major undertaking in and of itself - my fear is that changing this might cause as many regressions as it fixes things. This may need to be opened as a new issue.